src/main/java/io/openliberty/tools/intellij/runConfiguration/LibertyRunManagerListener.java

```diff
@@ -40,8 +41,11 @@ public void runConfigurationRemoved(@NotNull RunnerAndConfigurationSettings sett
             try {
                 VirtualFile vBuildFile = VfsUtil.findFile(Paths.get(runConfig.getBuildFile()), true);
                 LibertyModule libertyModule = libertyModules.getLibertyModule(vBuildFile);
-                if (libertyModule != null && libertyModule.getCustomStartParams().equals(runConfig.getParams())) {
-                    libertyModule.setCustomStartParams(""); // clear the custom params
+                if (libertyModule != null) {
+                    libertyModule.setRunInContainer(false); // clear the run in container setting for the Liberty module.
```

Comment:
The change to `run in container` should only happen if the parameters are equal. Move this line after the `setCustomStartParams("")` line, inside the `if`.

Reply:
Looks good.